src/screen.rs

```diff
@@ -832,6 +832,7 @@
         // Output the left prompt if it has changed.
         if left_prompt != zelf.actual_left_prompt {
             zelf.r#move(0, 0);
+            zelf.write_mbs_if_some(&term.and_then(|term| term.prompt_start.as_ref()));
```

Comment:
How is this feature used by terminals?
If it's just about not overwriting output, we should do it for right prompts too.
If it's meant to allow jumping between prompts, then the right prompt shouldn't get it.

Reply:
as for kitty, it only needs marking of

- start of PS1
- start of PS2
- start of application output

https://sw.kovidgoyal.net/kitty/shell-integration/#notes-for-shell-developers